scripts/run_integration_tests.sh

```diff
@@ -55,25 +72,40 @@ input_event_files=$(ls ./input_events)
 # Sort event files by name so that snapshots stay consistent
 input_event_files=($(for file_name in ${input_event_files[@]}; do echo $file_name; done | sort))
 
-# Generate a random 8-character ID to avoid collisions with other runs
-run_id=$(xxd -l 4 -c 4 -p < /dev/random)
 
-# Always remove the stack before exiting, no matter what
+
+# Always remove the stacks before exiting, no matter what
 function remove_stack() {
-    echo "Removing functions"
-    serverless remove --stage $run_id
+    for runtime in "${RUNTIMES[@]}"; do
```

Comment:
I think it's a bit odd to say `runtime in "${RUNTIMES[@]}"` here because each item in that list contains information other than the runtime (like the run ID). In other words, I think it's odd to say `run_id=$runtime[2]`. Maybe there is a better possible name for this, like `parameters_set in ${PARAMETERS_SETS[@]}`. Then we could say `run_id=$parameters_set[2]`.

Reply:
✅